apache-example.conf

```diff
@@ -0,0 +1,39 @@
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+ ServerName  my.server.name
+ ServerAlias  my.server.name
+ DocumentRoot /path/to/server-skeleton/
+
+ # Line below assumes that the pytho virtual environment is set up in server-skeleton/.venv
+ WSGIDaemonProcess etes python-path=/path/to/server-skeleton/ python-home=/path/to/server-skeleton/.venv
+ WSGIProcessGroup etes
+ WSGIScriptAlias / /path/to/server-skeleton/etesync_server/wsgi.py
+ WSGIPassAuthorization On
+
+ # To prevent any permission issues, it is best that the entire folder
+ # is chowned by the same group (www-data) that apache runs under. So:
+ # sudo chown -R :www-data /path/to/server-skeleton
+ 
+ # Also apache needs write access to the database file. So:
+ # sudo chmod 664 /path/to/server-skeleton/db.sqlite3
+ 
+ <directory /path/to/server-skeleton/>
```

Comment:
This looks wrong. You should be running "./manage.py collectstatic" and serve static files from the generated directory. Not just serve the source code.